providers/os/resources/smbios/aix.go

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package smbios
+
+import (
+	"bufio"
+	"errors"
+	"io"
+	"strings"
+
+	"go.mondoo.com/cnquery/v11/providers/os/connection/shared"
+)
+
+// AIXSmbiosManager gets system information from AIX systems.
+// smbios is not a thing on AIX, but we implement this interface
+type AIXSmbiosManager struct {
+	provider shared.Connection
+}
+
+func (s *AIXSmbiosManager) Name() string {
+	return "AIX Smbios Manager"
+}
+
+func (s *AIXSmbiosManager) Info() (*SmBiosInfo, error) {
+
+	cmd, err := s.provider.RunCommand("prtconf")
+	if err != nil {
+		return nil, err
+	}
+	if cmd.ExitStatus != 0 {
+		stderr, err := io.ReadAll(cmd.Stderr)
+		if err != nil {
+			return nil, err
+		}
+		return nil, errors.New("failed to run prtconf: " + string(stderr))
+	}
+
+	baseBoardInfo, sysInfo, err := ParsePrtConf(cmd.Stdout)
+	if err != nil {
+		return nil, err
+	}
+
+	return &SmBiosInfo{
+		BaseBoardInfo: baseBoardInfo,
+		SysInfo:       sysInfo,
+	}, nil
+}
+
+func ParsePrtConf(reader io.Reader) (BaseBoardInfo, SysInfo, error) {
+	baseBoardInfo := BaseBoardInfo{
+		Vendor: "IBM",
+	}
+	sysInfo := SysInfo{
+		Vendor: "IBM",
+		Family: "IBM Power Systems",
+	}
+
+	// Read each line from the reader
+	scanner := bufio.NewScanner(reader)
+	for scanner.Scan() {
+		line := scanner.Text()
+		// Process the line to extract relevant information
+		if strings.Contains(line, "System Model") {
+			baseBoardInfo.Model = strings.TrimSpace(strings.Split(line, ":")[1])
+			sysInfo.Model = baseBoardInfo.Model
+		} else if strings.Contains(line, "Machine Serial Number") {
+			baseBoardInfo.SerialNumber = strings.TrimSpace(strings.Split(line, ":")[1])
+			sysInfo.SerialNumber = baseBoardInfo.SerialNumber
+		} else if strings.Contains(line, "Firmware Version") {
+			baseBoardInfo.Version = strings.TrimSpace(strings.Split(line, ":")[1])
```

Comment:
All of these splits assume that there will always be a `:`, which most likely will be there but if not, we will panic.

Could we create a simple helper function that does this splitting and trimming safely?